Tests/SWBWebAssemblyPlatformTests/SWBWebAssemblyPlatformTests.swift

```diff
@@ -0,0 +1,119 @@
+// This source file is part of the Swift open source project
+//
+// Copyright (c) 2025 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Testing
+import SWBProtocol
+import SWBTestSupport
+import SWBTaskExecution
+import SWBUtil
+
+@Suite
+fileprivate struct SWBWebAssemblyPlatformTests: CoreBasedTests {
+    @Test(.requireThreadSafeWorkingDirectory, .enabled(if: getEnvironmentVariable("WASM_SDKROOT") != nil), arguments: ["wasm32"])
+    func wasiCommand(arch: String) async throws {
+        guard let sdkroot = getEnvironmentVariable("WASM_SDKROOT") else { return }
+        guard let toolchain = getEnvironmentVariable("WASM_TOOLCHAINS") else { return }
+
+        try await withTemporaryDirectory { (tmpDir: Path) in
+            let testProject = TestProject(
+                "TestProject",
+                sourceRoot: tmpDir,
+                groupTree: TestGroup(
+                    "SomeFiles",
+                    children: [
+                        TestFile("main.swift"),
+                        TestFile("static.swift"),
+                    ]),
+                buildConfigurations: [
+                    TestBuildConfiguration("Debug", buildSettings: [
+                        "ARCHS": arch,
+                        "CODE_SIGNING_ALLOWED": "NO",
+                        "DEFINES_MODULE": "YES",
+                        "PRODUCT_NAME": "$(TARGET_NAME)",
+                        "SDKROOT": sdkroot,
+                        "SWIFT_VERSION": "6.0",
+                        "SUPPORTED_PLATFORMS": "webassembly",
+                        "TOOLCHAINS": toolchain,
+                    ])
+                ],
+                targets: [
+                    TestStandardTarget(
+                        "tool",
+                        type: .commandLineTool,
+                        buildConfigurations: [
+                            TestBuildConfiguration("Debug", buildSettings: [:])
+                        ],
+                        buildPhases: [
+                            TestSourcesBuildPhase(["main.swift"]),
+                            TestFrameworksBuildPhase([
+                                TestBuildFile(.target("staticlib")),
+                            ])
+                        ],
+                        dependencies: [
+                            "staticlib",
+                        ]
+                    ),
+                    TestStandardTarget(
+                        "staticlib",
+                        type: .staticLibrary,
+                        buildConfigurations: [
+                            TestBuildConfiguration("Debug", buildSettings: [
+                                // FIXME: Find a way to make these default
```

Comment:
WebAssembly should be able to provide spec files for the library product types which override the generic copies and give this a default value. But I think that would be fine to do as a follow up later